addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -185,18 +189,18 @@ export class AnalyticDistribution extends Component {
     }
 
     // Autocomplete
-    get sourcesAnalyticAccount() {
-        return [this.optionsSourceAnalytic];
+    sourcesAnalyticAccount(groupId) {
```

Comment:
If you don't want all those `groupId` wouldn't it be possible/cleaner to have a sub-component per group/plan?

Maybe not to do now, just an idea

Reply:
that will be similar to what I had... a variable called `activeGroup` which changed when you focused to another plan.
As a sub-component, it would be an unchanging prop. 
Will probably be cleaner though.

Reply:
considerations:
1. changes to a tag (child component) will need to update the object in the parent component (will need a callback function)
2. managing focus from 1 plan to the next
Will explore in the next PR